diff --git a/node_modules/eth-json-rpc-infura/src/index.js b/node_modules/eth-json-rpc-infura/src/index.js
index 72fbdd4..fa1b836 100644
--- a/node_modules/eth-json-rpc-infura/src/index.js
+++ b/node_modules/eth-json-rpc-infura/src/index.js
@@ -1,6 +1,6 @@
 const createAsyncMiddleware = require('json-rpc-engine/src/createAsyncMiddleware')
 const { ethErrors } = require('eth-rpc-errors')
-const fetch = require('node-fetch')
+const fetch = require('node-fetch').default
 
 const RETRIABLE_ERRORS = [
   // ignore server overload errors
